Src/libCZI/Doc/CZICmd_usage.markdown

```diff
@@ -0,0 +1,635 @@
+CZIcmd Documentation                 {#czicmd}
+=======================
+
+The console application "CZIcmd" is provided for two purposes:
+- demonstrate the usage of the libCZI
+- it might be useful to convert/extract data from a CZI-file
+
+The synopsis of the program is:
+
+		Usage: CZIcmd.exe [OPTIONS]
+
+			using libCZI version 0.46
+
+		Options:
+			-h,--help         Print this help message and exit
+
+			-c,--command COMMAND
```

Comment:
Is it on purpose that only a subset of all available commands here is mentioned explicitely in the sections below?

![image](https://user-images.githubusercontent.com/9250590/208236862-3cd50729-8a87-49b0-9c67-b2a21d0cf17b.png)

Reply:
The rest looks good to me.

Reply:
> Is it on purpose that only a subset of all available commands here is mentioned explicitely in the sections below?
> 
> ![image](https://user-images.githubusercontent.com/9250590/208236862-3cd50729-8a87-49b0-9c67-b2a21d0cf17b.png)

-> #22